python/GafferImageTest/ImageTransformTest.py

```diff
@@ -171,10 +171,10 @@ def testDisabled( self ) :
 			t["transform"]["translate"].setValue( IECore.V2f( 2., 2. ) )
 			t["transform"]["rotate"].setValue( 90 )
 			t["enabled"].setValue( True )
-			self.assertNotEqual( r["out"].hash(), t["out"].hash() )
+			#self.assertNotEqual( r["out"].hash(), t["out"].hash() )
```

Comment:
This was part of my commit "DONT MERGE : For illustration, this is how I was locating code that evaluated global plugs in a non-global context", which added asserts to ensure that global plugs were forced to never be called in non-global contexts.  By explicitly setting the tile origin, and then hashing the image plug ( which implicitly hashes all it's children including the global plugs ), this test violates that rule.

I was kind of guessing that you wouldn't want those asserts anyways, I was thinking that this would be an optimization we would do only when it was profiled to matter.  But if you're mostly OK with this batch of commits, maybe we should think about keeping these asserts in there?

Failing to use global scope for global plugs is a performance hazard which may not show up in most testing but has the potential to dramatically slow down users with obscure and complex setups.

As for this test, do you think there's a good reason why it explicitly sets tileOrigin and channel and calls ImagePlug.hash() instead of calling ImagePlug.imageHash()?  And actually, what's it doing with CapturingSlot that it doesn't appear to actually use?